### PR TITLE
dont install debian recommends packages

### PR DESCRIPTION
**Pull request by pad92**

### CODE REVIEW

**Comment by rockandska** [comment]
Thanks @pad92 

Does it solve #4 or not related ?

Regards,

**Comment by DisasteR** [comment]
@rockandska yes it solve #4

File: tasks/install_Debian.yml [hunk 1]
```diff
@@ -31,5 +31,6 @@
 - name: "Install Erlang [Debian/Ubuntu]"
   apt:
     package:
-      - erlang
+      - erlang-nox
     state: present
+    install_recommends: no
```
Comment (rockandska) [inline tasks/install_Debian.yml]: could you please rebase with master and/or fix linting error by changing to `false` 
thanks

Comment (rockandska) [reply tasks/install_Debian.yml]: if you don't have the time to do it, let me access the [branch concerned by this PR](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) and I will do it.

**Comment by DisasteR** [comment]
@rockandska any news about that one ?

**Comment by rockandska** [comment]
> @rockandska any news about that one ?

Asked changed who was never addressed by @pad92

**Comment by DisasteR** [comment]
> Asked changed who was never addressed by @pad92

what ?

**Comment by rockandska** [comment]
> > Asked changed who was never addressed by @pad92
> 
> what ?

image

File: tasks/install_Debian.yml (hunk 1 above)
Comment (rockandska) [reply tasks/install_Debian.yml]: Waiting for this to merge

Edit: ok, never finished the review so It was not displayed.....

**Review by rockandska — CHANGES_REQUESTED** [review]

File: tasks/install_Debian.yml (hunk 1 above)
Comment (DisasteR) [reply tasks/install_Debian.yml]: done

**Review by rockandska — APPROVED** [review]

**Pull request merged**